pages/vsa/design.md

```diff
@@ -17,6 +17,8 @@ permalink: false
 # Excludes this page from the sitemap. This isn't really important because we aren't publishing to prod anyway,
 # but still good to do.
 private: true
+
+includeBreadcrumbs: false
```

Comment:
@sshein fysa this changed the breadcrumbs to look like 

![image](https://user-images.githubusercontent.com/1915775/107414919-a944a980-6ae0-11eb-944b-fefecc15f53d.png)

Reply:
oh I think that was an accident! didn't mean to touch the breadcrumbs

Reply:
oh wait this is something you did haha I'm not paying enough attention

Reply:
haha previously it had a "VSA" breadcrumb there that led to a page that didn't exist. My change just removed that crumb.